packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -190,7 +189,7 @@ class TestCommand extends FlutterCommand {
     }
     testArgs.addAll(files);
 
-    final String shellPath = tools.getHostToolPath(HostTool.SkyShell) ?? platform.environment['SKY_SHELL'];
```

Comment:
Is it important to keep the old behavior of falling back to look up the path from a SKY_SHELL env var?